LearnosityDemo/Pages/AuthorAPIDemo.cshtml.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Text;
+using LearnositySDK.Request;
+using LearnositySDK.Utils;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Newtonsoft.Json;
+
+namespace LearnosityDemo.Pages
+{
+    public class AuthorAPIDemoModel : PageModel
+    {
+        public void OnGet(string mode)
+        {
+            // Author API contains 2 modes
+            // item_edit mode and item_list mode
+            // the below example generate initOptions for item_edit mode
+            // more information about item_list mode can be found in initializeItemList() function
+            Init init = (mode == "item_list") ? initializeItemEdit() : initializeItemList();
```

Comment:
is this conditional correct? if mode is item_list, then you will return itemEdit, and if mode is not item_list then you return item_list?

Reply:
ohh I need to correct that , If mode is Item list it should return itemList not item edit. Thanks For pointing it out I will make that change